### PR TITLE
[RM-1594] - python-domino release-1.1.0

### PR DESCRIPTION
**Pull request by ddl-olsonJD**
## Link to JIRA

[DOM-RM-1594](https://dominodatalab.atlassian.net/browse/RM-1594)

## Description
Updated python domino version from 1.0.8 to 1.1.0

### Added
* Added black and flake8 formatting
* Implemented pre-commit for black / flake8 formatting.
* Added / updated dataset endpoint usability.
* Added option to bypass certification verification
* Added tags functionality

### Changed
* updated airflow dependency to 2.2.4
* Updated runs_stdout output
* Fixed issues with import from domino_data
* Updated nbconvert dependency version to 6.3.0
* updated request user-agent to python-domino/{version}

### Pull Request Reminders

- [x] Has relevant documentation been updated?
- [x] Does the code follow [Python Style Guide] (https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html)
- [x] Update the [changelog](https://github.com/dominodatalab/python-domino/blob/master/CHANGELOG.md)
- [x] Are the existing unit tests still passing?
- [ ] Have new unit tests been added to cover any changes to the code?
- [x] Has the JIRA ticket(s) been linked above?

### CODE REVIEW

File: domino/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "1.0.8"
+__version__ = "1.0.9"
```
Comment (ddl-awroblicky) [inline domino/_version.py:1]: should this be 1.1.0?

**Comment by dmitriyvolk** [comment]
What triggered the minor version bump? Any particular incompatibility with the previous one?

**Comment by ddl-olsonJD** [comment]
> What triggered the minor version bump? Any particular incompatibility with the previous one?

There are no incompatibility changes, we are just moving towards are more standard versioning:
See https://dominodatalab.atlassian.net/wiki/spaces/~62671dd0593e80006f14ef0c/pages/2298511380/python-domino+semantic+versioning+proposal

**Comment by dmitriyvolk** [comment]
So you're basically starting with a clean slate at 1.1.0? Sounds good

**Review by dmitriyvolk — APPROVED** [review]

**Pull request merged**